stories/type-scale.stories.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Type Scale stories.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Cell, Grid, Row } from '../assets/js/material-components';
+import Widget from '../assets/js/googlesitekit/widgets/components/Widget';
+import { Fragment } from '@wordpress/element';
+
+import './type-scale.scss';
+
+function TypeScaleWidget( { name, label, sizes } ) {
+	return (
+		<Cell size={ 6 }>
+			<Widget
+				className={ `googlesitekit-type-scale googlesitekit-type-scale-${ name }` }
+			>
+				<h2>{ label }</h2>
+				{ Object.keys( sizes ).map( ( key, index ) => {
+					return (
+						<p
+							key={ index }
+							className={ `googlesitekit-type-scale-${ name } ${ key }` }
+						>
+							{ sizes[ key ] }
+						</p>
+					);
+				} ) }
+			</Widget>
+		</Cell>
+	);
+}
+
+const Template = () => (
+	<Fragment>
+		<Grid className="googlesitekit-widget-area googlesitekit-widget-area--boxes">
+			<Row className="googlesitekit-widget-area-widgets">
+				<Cell size={ 12 }>
+					<h1>Type Scale</h1>
+				</Cell>
+				<TypeScaleWidget
+					name="display"
+					label="Display"
+					sizes={ {
+						lg: 'Large 58/64',
+						md: 'Medium 46/52',
+						sm: 'Small 38/44',
+					} }
+				/>
+				<TypeScaleWidget
+					name="headline"
+					label="Headline"
+					sizes={ {
+						lg: 'Large 32/40',
+						md: 'Medium 28/36',
+						sm: 'Small 22/28',
+					} }
+				/>
+				<TypeScaleWidget
+					name="title"
+					label="Title"
+					sizes={ {
+						lg: 'Large 18/24',
+						md: 'Medium 16/20',
```

Comment:
As mentioned there's a discrepancy on Figma that we need to clarify regarding this medium line height value, this is probably OK but we may need to revisit depending on Seba's response.